projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -385,7 +385,7 @@ public function build_wpcom_masterbar( $wp_admin_bar ) {
 		$this->add_reader_submenu( $wp_admin_bar );
 
 		// Right part.
-		if ( Jetpack::is_module_active( 'notes' ) ) {
+		if ( Jetpack::is_module_active( 'notes' ) && ! Jetpack_Notifications::is_block_editor() ) {
```

Comment:
It should be `\Jetpack_Notifications` otherwise it's causing a fatal.

Reply:
thanks for catching that!